app/views/devise/mailer/password_change.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= render 'shared/email_header', preheader: 'We&#39;re contacting you to notify you that your password has been changed. To report any issues or for general inquiries email us at hello@eruditionapp.com. Erudition App, Inc. 520 Chestnut St, Philadelphia, PA 19106' %>
+
```

Comment:
let's be sure to update hello@eruditionapp.com to whatever our support contact becomes